saleor/graphql/giftcard/schema.py

```diff
@@ -0,0 +1,22 @@
+import graphene
+from graphql_jwt.decorators import permission_required
+
+from ...giftcard import models
+from ..core.fields import PrefetchingConnectionField
+from .resolvers import resolve_gift_card
+from .types import GiftCard
+
+
+class GiftCardQueries(graphene.ObjectType):
+    gift_card = graphene.Field(
+        GiftCard, id=graphene.Argument(graphene.ID, required=True),
+        description='Lookup a gift card by ID.')
+    gift_cards = PrefetchingConnectionField(
+        GiftCard, description='List of gift cards')
```

Comment:
```suggestion
        GiftCard, description='List gift cards')
```

Reply:
I think my version is correct according to this page. 
https://www.dictionary.com/browse/list

Reply:
Oh my bad. True, the context is "It's a list of gift cards". I read it "It lists the gift cards".